lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3916,7 +3916,10 @@ void SwiftASTContext::ValidateSectionModules(
 
   Status error;
 
-  Progress progress("Loading Swift module", module_names.size());
+  Progress progress(
+      llvm::formatv("Loading Swift module '{0}'. Submodule",
```

Comment:
Sorry for going back and forth on this but "swift module <modulename> submodule" sounds too repetitive. If we retitle it this way it addresses the repetitiveness.

```suggestion
      llvm::formatv("Loading Swift module '{0}' dependencies",
```

Reply:
This is much better wording, I'll change the title here